### PR TITLE
networks update 19.03.21

### PR DESCRIPTION
**Pull request by itzikhillel**

### CODE REVIEW

File: FairBid/tapjoy.plist.xml [hunk 1]
```diff
@@ -28,6 +28,290 @@
...
 			</dict>
+			<dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>hs6bdukanm.skadnetwork</string>
+			</dict> 
+			 <dict>
+				<key>SKAdNetworkIdentifier</key>
+				<string>22mmun2rn5.skadnetwork</string>
+			</dict>
+            <dict>
```
Comment (waqark1) [inline FairBid/tapjoy.plist.xml:39]: The indentation is incorrect. We should keep the indentation of the files according to xml standard.

**Review by waqark1 — CHANGES_REQUESTED** [review]
please fix the indentation of the files.

File: FairBid/tapjoy.plist.xml [hunk 2]
```diff
@@ -28,6 +28,290 @@
...
+				<key>SKAdNetworkIdentifier</key>
+				<string>ejvt5qm6ak.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>cg4yq2srnc.skadnetwork</string>
+			</dict>
+            <dict>  
+				<key>SKAdNetworkIdentifier</key>
+				<string>.skadnetwork</string>
```
Comment (thomasmso) [inline FairBid/tapjoy.plist.xml:209]: Pretty sure this is incorrect :)

Comment (waqark1) [reply FairBid/tapjoy.plist.xml:209]: Thank you for pointing it out @thomasmso :)

**Review by waqark1 — APPROVED** [review]
I updated the files and fixed the mistakes and applied changes according to the comments.

**Comment by waqark1** [comment]
Branch deleted after Merge to master

**Pull request merged**